_assets/patches/geth/0019-whisperv6-send-self-messages-without-subscribe.patch

```diff
@@ -0,0 +1,58 @@
+diff --git a/whisper/whisperv6/api.go b/whisper/whisperv6/api.go
+index 16db034e..9d5ebf84 100644
+--- a/whisper/whisperv6/api.go
++++ b/whisper/whisperv6/api.go
+@@ -264,6 +264,7 @@ func (api *PublicWhisperAPI) Post(ctx context.Context, req NewMessage) (bool, er
+ 		WorkTime: req.PowTime,
+ 		PoW:      req.PowTarget,
+ 		Topic:    req.Topic,
++		self:     true,
```

Comment:
would prefer not to change envelope struct and do it in a way similar to https://github.com/ethereum/go-ethereum/pull/16172/files

Reply:
We can either change struct or add a new parameter into `add()` method. For me, the first way is cleaner.

Reply:
1. If I add a new parameter in add() we'll have a conflict with current Ethereum.
2. The add method call will look like `add(env, true, false)` for light client forwarded message. For me, it's not very clear to have >2 parameters or, worse, boolean flags.
@themue What do you think about it?

Reply:
To me it looks like a property of an envelope, so adding it as field IMHO makes more sense. But I would like to have a name more describing the intention of this flag.

Reply:
sure, i am not insisting on changing this. however, my point is that it is easier to find a caller with a particular argument than to track all of the changes to a mutable object

Reply:
Hmm, good argument. Also setting it after creation with `NewEnvelope()` feels not optimal and the only reader is `add()`. Will take a look at your hint, @dshulyak .

Reply:
@JekaMas The `add(*Envelope, bool, bool)` surely has two boolean flags. But in definition of `add()` their intention is pretty clear by naming (like done with `isP2P`)  and for calling their could be used constants with speaking names.

Reply:
Ok, I'll add param `isForwarded` to add method.